internal/providers/terraform/azure/automation_account.go

```diff
@@ -0,0 +1,79 @@
+package azure
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/infracost/infracost/internal/schema"
+	"github.com/shopspring/decimal"
+	"github.com/tidwall/gjson"
+)
+
+func GetAzureRMAutomationAccountRegistryItem() *schema.RegistryItem {
+	return &schema.RegistryItem{
+		Name:  "azurerm_automation_account",
+		RFunc: NewAzureRMAutomationAccount,
+		ReferenceAttributes: []string{
+			"resource_group_name",
+		},
+	}
+}
+
+func NewAzureRMAutomationAccount(d *schema.ResourceData, u *schema.UsageData) *schema.Resource {
+	var monthlyJobRunMins, nonAzureConfigNodeCount, monthlyWatcherHours *decimal.Decimal
+	group := d.References("resource_group_name")[0]
+	location := strings.ToLower(group.Get("location").String())
+	costComponents := make([]*schema.CostComponent, 0)
+
+	if u != nil && u.Get("monthly_job_run_mins").Type != gjson.Null {
+		monthlyJobRunMins = decimalPtr(decimal.NewFromInt(u.Get("monthly_job_run_mins").Int()))
+		if monthlyJobRunMins.IsPositive() {
+			costComponents = append(costComponents, runTimeCostComponent(location, "500", "Basic Runtime", monthlyJobRunMins))
+		}
+	} else {
+		var unknown *decimal.Decimal
+		costComponents = append(costComponents, runTimeCostComponent(location, "500", "Basic Runtime", unknown))
```

Comment:
You can use existing variable here (and all the cases below), not unknown
```suggestion
              costComponents = append(costComponents, runTimeCostComponent(location, "500", "Basic Runtime", monthlyJobRunMins))
```